### DISCUSSION TITLE
Change performance data units

### DISCUSSION THREAD

**Discussion by heinowalther** [sni/thruk#1089] [Q&A]
Hi there
I am using Thruk via the omdist, so I am using InfluxDB and Grafana for graphing of performance data.
I am looking for a way to show more useful output in both the graphs, but also in the Thruk view.
The issue is with some custom checks that output seconds or bytes.
In the case of seconds, I would like to be able to re-calculate this into hours or days, and similar with the bytes, I would very much like to be able to report MB/GB/TB, simply because the numbers become more or less useless when we are talking TB's shown as bytes ;-) as well as days shown as seconds.  Also on most graphs the numbers become so long that they do no longer fit the view...

Of cause the creators of the checks points to the fact that the system they are checking reports in seconds and bytes, and recommend to "fix" this in the presentation layer.
But is there any way in Thruk where I can alter this per service?  (I haven't found it yet)
My only way to "fix" this would be to write a wrapper that converts the performance output, but I don't think it is the correct solution to this problem...

Other suggestions are very welcome :-)

**Comment by dan-m-joh** [comment]
What are you using for "graphing"?
We are using a similar setup and are using "histou" to create the graphs in Thruk and it automagically converts seconds to days/weeks/...

![image](https://user-images.githubusercontent.com/26163222/114179301-176de700-993f-11eb-9e04-aa876e0e3b56.png)

The same is true for B/KB/MB/... (or B/KiB/MiB/...) as well.

Regards,
D/\N

**Comment by heinowalther** [comment]
Well here is an example...
<img width="695" alt="Screenshot 2021-04-09 at 14 30 14" src="https://user-images.githubusercontent.com/6001138/114180243-3c168e80-9940-11eb-83da-257b78198ba2.png">

**Comment by dan-m-joh** [comment]
And here is how it looks for my Uptime-Check..

![image](https://user-images.githubusercontent.com/26163222/114180879-1e95f480-9941-11eb-910d-38e11bc547fe.png)
